.ci-config/rippled.cfg

```diff
@@ -78,35 +54,10 @@ validators.txt
 [rpc_startup]
 { "command": "log_level", "severity": "info" }
 
-# severity (order: lots of information .. only errors)
-# debug
-# info
-# warn
-# error
-# fatal
-
 [ssl_verify]
 1
 
-# The [features] stanza does not currently work for standalone mode: https://github.com/XRPLF/xrpl-dev-portal/issues/1762#issuecomment-1441252450
-
-
-# In order to enable an amendment which by default would vote "No", you must include its amendment id and name here.
-# To add amendments specifically from the latest releases of rippled:
-# 1. Go to https://xrpl.org/known-amendments.html
-# 2. Find the first amendment in the latest releases of rippled which are not already in the list below
-# 3. Click on each amendment to get their Amendment ID and name to add to this list manually.
-#    You will likely update the list with all amendments from a new release of rippled all at once.
-
-# To get the list of amendments on a network (e.g. devnet) follow the steps in xrpl.js's CONTRIBUTING.md for "Updating the Docker container".
-# https://github.com/XRPLF/xrpl.js/blob/main/CONTRIBUTING.md
-# (Running the script `getNewAmendments.js` should help you identify any new amendments that should be added.)
-#
-# Note: The version of rippled you use this config with must have an implementation for the amendments you attempt to enable or it will crash.
-# If you need the version of rippled to be more up to date, you may need to make a comment on this repo: https://github.com/WietseWind/docker-rippled
-
 [features]
-# Devnet amendments as of June 28th, 2023
```

Comment:
I would rather the comments in here are kept - it makes the long list of features easy to read

Reply:
good catch, not sure why those got removed on compile/build. Will add them back.